src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -366,7 +380,9 @@ export class SummaryPanelNode extends React.Component<SummaryPanelNodeProps, Sum
 }
 
 const mapStateToProps = (state: KialiAppState) => ({
-  jaegerState: state.jaegerState
+  jaegerState: state.jaegerState,
+  rank: state.graph.toolbarState.rank,
+  rankBy: rankBySelector(state)
```

Comment:
I get why you need these both, because we don't clear `rankBy` even if we disable `rank`, right?   But here's my suggestion, add a new redux prop, something like `minRank`.  This would be 0 when `rank` is disabled.  When `rank` is enabled it would store the minRank, something in the [1..100] range.   If we had this it would be the only redux prop needed here, as it would get reset each time ranks were calculated, or when `rank` was disabled.  Then you'd be able to show the current rank range as well as the individual node rank.

Reply:
Adding `minRank` as a redux prop sounds good but we'd still probably need to subscribe to `rankBy` updates since the `minRank` could stay the same for the overall graph even though the scoring criteria changes and the rank for that particular node changed. It'd be nice to subscribe to updates to a particular node and re-render the component when any of the data changes but iirc the graph updates based on the layout changing and since node scoring is a sort of post processing, it doesn't affect the layout.

Reply:
Yes, you're right if minRank is just an int.  I suppose we could consolidate things into RankInfo, and it could be reassigned with a new object on any change.  Depends on how much you don't want to use rankBy just as a refresher.

Reply:
Consolidating things into a single `RankInfo` object seems like a good approach. It might cause unnecessary renders later for components that just want to subscribe to either `rank` or `rankBy` but imo that's a pretty minor concern. If the node itself was in the store I think the most direct option would be to subscribe to the node by it's node id.

Reply:
The summary target rank/score was not being updated when the graph was re-scored. That was the underlying issue requiring the `rankBy` hack. I updated this PR so that when the nodes are re-scored the summary target data will also get updated triggering a re-render for the node summary panel.